package.json

```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "@types/jest": "^21.1.2",
     "jest": "^21.2.1",
-    "rxjs": "5.4.3",
+    "rxjs": "^5.5.5",
```

Comment:
To use the lettable operators in the tests we need to bump this. If you'd rather not, we could alternative use `.call` directly with the older style operators, which isn't too crazy when there's only a single operator to apply

```diff
- const actual = hot(sourceTemplate).pipe(
-   tap(feedNextIntoRelay, feedErrorIntoRelay, feedCompleteIntoRelay)
- );

+ const actual = _do.call(hot(sourceTemplate, feedNextIntoRelay, feedErrorIntoRelay, feedCompleteIntoRelay)
```

But gets hairy when there are multiple operators applied. The current tests don't seem to apply more than one operator though.

```js
const items = filter.call(map.call(of(1, 2, 3), d => d * 1), d => d > 10);

// same as

let items = of(1, 2, 3);
items = map.call(items, d => d * 1);
items = filter.call(items, d => d > 10);
```

Reply:
Agree that we should bump this :).